src/popup/components/DetailsItem.vue

```diff
@@ -4,13 +4,15 @@
     :class="{
       expandable,
       expanded,
+      warning,
     }"
   >
     <Component
       :is="expandable ? 'BtnPlain' : 'div'"
       class="text-label label"
       @click="toggleExpanded()"
     >
+      <slot name="before-label" />
```

Comment:
I think this component's logic of inserting the label text, label slot and now before-label part is too complex. In my opinion this should be refactored to to something like this:
```
<span class="label">
  <slot name="label">{{ label }}</slot>
</span>
```